runners/local/progress.go

```diff
@@ -60,7 +60,16 @@ func (r *RunnerLocal) UpdateRunnerProgress() {
 	if r.runnerProgress.SyncState != iface.VerifySyncState && r.runnerProgress.SyncState != iface.CleanupSyncState { //XXX: if we're cleaning up or verifying, we don't want to overwrite the state
 		r.runnerProgress.SyncState = srcStatus.SyncState
 	}
-	r.runnerProgress.SrcAdditionalStateInfo = srcStatus.AdditionalInfo
+	stateInfo := ""
+	// if we're in change stream mode, but not all tasks have been fully completed, we need to keep the state as Initial Sync
+	if r.runnerProgress.SyncState == iface.ChangeStreamSyncState && !flowStatus.AllTasksCompleted {
```

Comment:
I think this can be consolidated with the previous if statement to more clear. right now it seems kinda difficult to follow by first maybe updating sync state and then making this change if another condition is true. Could we set this up to just be very explicit what we set sync state to?